configuration/concepts/lab_tests.csv

```diff
@@ -95,14 +100,56 @@ d3966267-5882-437b-ae14-18dee9657d14,,,RBCs,Hematíes,Hematíes,RBCs,,LabTest,Co
 1b985633-13de-4afa-828e-ef38721c660e,,,Ferning,Helecho,Helecho,Ferning,,LabTest,Coded,,Positive; Negative,,,,,,,,
 dbb2a948-ab68-4474-a24b-5f0ecf697f88,,,Whiff Test,Prueba de olor,Prueba de olor,Whiff Test,,LabTest,Coded,,Positive; Negative,,,,,,,,
 acdc2f09-2844-43e0-9067-8e71da8319ae,,,Urinalysis,Análisis de orina,Análisis de orina,Urinalysis,,LabSet,N/A,,,PH; Specific gravity; Glucose; Ketones; Urobilinogen; Nitrites; Leukocyte esterase; Bilirubin; Blood in urine; Protein,,,,,,,
-4bb7df94-7b92-4e5f-8221-064a76d65e17,,,Yeast,Hongos,Hongos,Yeast,,LabTest,Coded,,Positive; Negative;,,,,,,,,
-b2a75309-4eb5-414d-9f29-8ecb4ac7dd32,,,Bacterial vaginosis,Vaginosis bacteriana,Vaginosis bacteriana,Bacterial vaginosis,,LabTest,Coded,,Positive; Negative;,,,,,,,,
-74ab27c4-c745-45ff-b7e6-e8dbaf933458,,,Trichomonas,Trichomonas,Trichomonas,Trichomonas,,LabTest,Coded,,Positive; Negative;,,,,,,,,
+4bb7df94-7b92-4e5f-8221-064a76d65e17,,,Yeast,Hongos,Hongos,Yeast,,LabTest,Coded,,Positive; Negative,,,,,,,,
+b2a75309-4eb5-414d-9f29-8ecb4ac7dd32,,,Bacterial vaginosis,Vaginosis bacteriana,Vaginosis bacteriana,Bacterial vaginosis,,LabTest,Coded,,Positive; Negative,,,,,,,,
+74ab27c4-c745-45ff-b7e6-e8dbaf933458,,,Trichomonas,Trichomonas,Trichomonas,Trichomonas,,LabTest,Coded,,Positive; Negative,,,,,,,,
 14e99cf4-d294-4905-8837-40e704d8a6ad,,,Vaginal Candidiasis,Candidiasis vaginal,Candidiasis vaginal,Vaginal Candidiasis,,LabTest,Text,,,,,,,,,,
-ab6588f6-87ee-45ca-a6c5-dc1337f83469,,,Gonorrhea,Gonorrea,Gonorrea,Gonorrhea,,LabTest,Coded,,Positive; Negative;,,,,,,,,
-637b4f48-8717-45e2-a5f8-96869e3bb768,,,Chlamydia,Clamidia,Clamidia,Chlamydia,,LabTest,Coded,,Positive; Negative;,,,,,,,,
+ab6588f6-87ee-45ca-a6c5-dc1337f83469,,,Gonorrhea,Gonorrea,Gonorrea,Gonorrhea,,LabTest,Coded,,Positive; Negative,,,,,,,,
+637b4f48-8717-45e2-a5f8-96869e3bb768,,,Chlamydia,Clamidia,Clamidia,Chlamydia,,LabTest,Coded,,Positive; Negative,,,,,,,,
+ff74d247-aa76-4f2c-bcfd-01be27559681,,,ABO,ABO,ABO,ABO,,LabTest,Coded,,A;B;AB;O,,,,,,,,
+9b51c857-2d8e-4ef9-bece-a80cbd175ae6,,,Rhesus Factor,factor rhesus,RH,RH,,LabTest,Coded,,Positive; Negative,,,,,,,,
+d3b33cb4-4f6a-44e3-abd5-82e794d17ef7,,,Blood Grouping,Agrupación Sanguínea,Agrupación Sanguínea,Blood Grouping,,LabSet,N/A,,,ABO;Rhesus Factor,,,,,,,
+a5bee8bd-530a-4d5f-bf34-27f55b3b2515,,,Major,Principal,Principal,Major,,LabTest,Text,,,,,,,,,,
+cf971905-bd1e-4901-8633-a77f5263a71c,,,Minor,Menor,Menor,Minor,,LabTest,Text,,,,,,,,,,

```

Comment:
We change these to **Major Crossmatch** and **Minor Crossmatch**

Reply:
Done